packages/p2p/src/components/my-ads/filter-payment-methods/filter-payment-methods.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import debounce from 'lodash.debounce';
+import { SearchBox } from '@deriv/components';
+import { observer } from '@deriv/stores';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import FilterPaymentMethodsResults from './filter-payment-method-results';
+
+// TODO: Merge this component with FilterModal once navigation flow is fixed
+const FilterPaymentMethods = observer(({ selected_methods, setSelectedMethods }) => {
+    const { my_profile_store, my_ads_store } = useStores();
+    const filtered_payment_methods = my_profile_store.payment_methods_list.filter(
+        payment_method => !selected_methods.includes(payment_method.value)
+    );
```

Comment:
@nada there has been an ongoing discussion about this, i think we need to check if they want to keep pursuing the new navigation flow, this was merged from my other PR

Reply:
if not, ill move this only under quick add modal, since i initially left it out as a reusable component